Tools/Scripts/webkitpy/port/driver.py

```diff
@@ -572,6 +573,8 @@ def cmd_line(self, pixel_tests, per_test_args):
             cmd.append('--show-window')
         if self._port.get_option('accessibility_isolated_tree'):
             cmd.append('--accessibility-isolated-tree')
+        if self._port.get_option('additional_header'):
+            cmd.append("--additional-header '%s'" % self._port.get_option('additional_header'))
```

Comment:
Given this is constructing a list to pass to `subprocess.Popen` or similar, don't we actually want:

```suggestion
            cmd.append("--additional-header")
            cmd.append(self._port.get_option('additional_header'))
```

(compare with the lines just below)

Reply:
> Given this is constructing a list to pass to `subprocess.Popen` or similar, don't we actually want:

After further testing, I see that this change is actually causing WebKitTestRunner to crash in the case where the
`--additional-header` argument has multiple values; for example: `--additional-header='foo=bar hoge=moge'` or whatever. So I’ll need to fix that before this can be merge-worthy

Reply:
> So I’ll need to fix that before this can be merge-worthy

Given this is now resolved, is that fixed?

Reply:
OK, I’ve fixed the crash. It turns out it’s not actually necessary to append `--additional-header` to the `cmd` value in that `cmd_line` function — the `--additional-header` option has the expected effect without even doing that.

As far as I could see, the only observable effect that appending `--additional-header` there has on anything it to cause `--additional-header` and its value to be appended to the `Command line:` info message that gets emitted by the `print_config` function in the `Printer` logger.

So I instead just directly added some additional info logging to that `print_config` function — to explicitly echo out the `--additional-header` value that’s set in the `self._options.additional_header` value.

And if that all now looks OK, then I think this is otherwise merge-worthy.